packages/flutter/lib/src/services/raw_keyboard_linux.dart

```diff
@@ -39,32 +39,30 @@ class RawKeyEventDataLinux extends RawKeyEventData {
   /// (GLFW, GTK, QT, etc) may have a different key code mapping.
   final KeyHelper keyHelper;
 
+  /// The name of the printable key. This is typically the character that [keyCode] would
+  /// produce without any modifier keys. For dead keys, it is typically the diacritic
+  /// it would add to a character. Defaults to an empty string, asserted to be not null.
+  final String codePoint;
```

Comment:
`codePoint` isn't a great name, since it's a string (and could contain multiple "code points").

I'd call it something like unicodeScalarValuesProduced.

You could also just leave it as an int, and pack the first two scalars into it.  Then it would represent exactly the amount of data we can accept. Definitely document it.

Reply:
Changed the name and documented that we handle two at most, but it's typically one value